### ISSUE TITLE
Filename/folder permissions bug when updating lemma text

### ISSUE THREAD

**Issue by susanodd** [Signbank/Global-signbank#1067]
For some reason, it's not possible to update the Lemma text to a different text that is not being used.
The reason? The code is trying to move the image file which is stored under "glossimage/acronym/AN/annotation_NNN.png" to a new name. (Although it's using the Lemma annotation, not the gloss annotation.)

(This is local.)

**Comment by susanodd** [comment]
Okay, I got it to allow me to update the (Dutch) Lemma text by changing the permissions on the folder of glossimage.

Now the "Gloss" image was moved to a new subdirectory of the first two characters of the Lemma text.

Is this correct? I thought we were using annotation in the filenames since more than one gloss can be in the lemma?

(This seems kind of like a side effect. Although I guess it's paired with the gloss id. Although now there is also an extra (wrong) image in the new location from when I typed it in with a "2" after it and it failed.)
